### PR TITLE
Adds automation of OWL format generation and deployment

### LINKED ISSUE #24: Translate ontology into OWL Functional Syntax with a workflow for generating RDF/XML
The authoritative development version of `./phyloref.owl` should be stored as an OWL Functional Syntax file and renamed to `./phyloref.ofn`. This will make it easier to compare diffs of this file when changing it during development.

The canonical version of this file (`/docs/phyloref.owl`) should remain in RDF/XML. We should create a workflow for generating this file from the OWL Functional Syntax development version at release time. Note that the OWLAPI default export in the OWL Functional Syntax doesn't include the version number of the sofware used to generate that file, so we will need to add that in manually.

### PR DESCRIPTION
**Pull request by hlapp**
This makes the assumption that the primary ontologies for which OWL format generation and deployment to web is desired are developed in OWL Functional Syntax format (.ofn). Hence, it won't do anything for ontologies (such as demo ontology files etc) that aren't in .ofn format, and it will at present only act on .ofn files in the root directory of the repo.

The main tool that is used for automatic annotation and conversion is [robot]. It needs to be installed before this Makefile will work. By default it is assumed to be in the ./bin directory, but the location
can easily be overridden by `make ROBOT="/path/to/robot"`.

Also provides targets for profile validation. This also depends on robot.

Closes #24.

[robot]: http://robot.obolibrary.org/

### CODE REVIEW

**Comment by hlapp** [comment]
@gaurav do you want to cast a second pair of eyes over this? It's fine if not, but if you do can you please assign yourself as a reviewer to signify that.

Otherwise I'll merge tonight or tomorrow.

File: Makefile
Comment (gaurav) [inline Makefile:4]: I think users are more likely to have `robot` in their `$PATH` than in a `bin/` subdirectory. I think we ought to default to setting `ROBOT = robot` and then let people override it in the command line as needed (i.e. `make ROBOT=./bin/robot`).

EDIT: I said command line instead of `$PATH` earlier.

**Review by gaurav — APPROVED** [review]
I'm not enough of a Makefile programmer to confirm that the code makes sense, but I have confirmed that all the targets work (except for the -tcan ones, since tcan won't be included until after PR #28 has been merged).

File: Makefile
Comment (hlapp) [reply Makefile:4]: Perhaps in theory. In practice there won't be many people running this for a while (mostly me, and perhaps you). More importantly it's geared towards eventually being run on Travis. Yes, we could add it to the `PATH` there, but that seems an unnecessary additional step?

**Pull request merged**
